benchmarks/scala-dotty/build.sbt

```diff
@@ -5,11 +5,11 @@ lazy val scalaDotty = (project in file("."))
     name := "scala-dotty",
     version := (version in renaissanceCore).value,
     organization := (organization in renaissanceCore).value,
-    scalafmtConfig := Some(file(".scalafmt.conf")),
-    scalaVersion := "2.12.8",
+    scalaVersion := "2.13.5",
+    scalacOptions += "-Ytasty-reader",
     libraryDependencies ++= Seq(
-      "commons-io" % "commons-io" % "2.6",
-      "ch.epfl.lamp" % "dotty-compiler_0.12" % "0.12.0"
+      "org.scala-lang.modules" %% "scala-collection-compat" % "2.4.2",
+      "org.scala-lang" % "scala3-compiler_3.0.0-RC1" % "3.0.0-RC1"
```

Comment:
I think I actually tried RC2, but the compilation fails, because (I think) the Tasty reader in Scala 2.13.5 is too old to read the jars produced by RC2. Actually, there is also RC3 on Maven central and it has the same problem, which is why I settled for RC1 back then. I think that we could wait for 3.0.0 (final) and hopefully, the updated Tasty reader will find its way into Scala 2.13.6.

Here's the error message I get with RC2 (and RC3):
```
[error] error while loading package, class file '/home/pallas/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3.0.0-RC2/3.0.0-RC2/scala3-compiler_3.0.0-RC2-3.0.0-RC2.jar(dotty/tools/package.class)' is broken
[error] (class scala.tools.tasty.UnpickleException/TASTy signature has wrong version.
[error]  expected: {majorVersion: 28, minorVersion: 0 [unstable release: 1]}
[error]  found   : {majorVersion: 28, minorVersion: 0 [unstable release: 3]}
[error] 
[error] This TASTy file was produced by an unstable release.
[error] To read this TASTy file, your tooling must be at the same version.
[error] The TASTy file was produced by Scala 3.0.0-RC2-nonbootstrapped.
[error] Note that your tooling is currently using an unstable TASTy version.)
[error] error while loading Main, class file '/home/pallas/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3.0.0-RC2/3.0.0-RC2/scala3-compiler_3.0.0-RC2-3.0.0-RC2.jar(dotty/tools/dotc/Main.class)' is broken
[error] (class scala.tools.tasty.UnpickleException/TASTy signature has wrong version.
[error]  expected: {majorVersion: 28, minorVersion: 0 [unstable release: 1]}
[error]  found   : {majorVersion: 28, minorVersion: 0 [unstable release: 3]}
[error] 
[error] This TASTy file was produced by an unstable release.
[error] To read this TASTy file, your tooling must be at the same version.
[error] The TASTy file was produced by Scala 3.0.0-RC2-nonbootstrapped.
[error] Note that your tooling is currently using an unstable TASTy version.)
```